src/components/DiagonalGradient.js

```diff
@@ -6,7 +6,7 @@ import { colors } from '../config';
 
 export const DiagonalGradient = ({
   children,
-  colors: gradientColors = [colors.primary, colors.secondary],
+  colors: gradientColors = [colors.primary, colors.primary],
```

Comment:
shouldn't this stay `secondary` to keep the buttons like in the screenshot ![image](https://github.com/user-attachments/assets/1cd53254-19d2-47a0-8576-99007121eca8)

Reply:
The screenshots were taken before the code was updated. I've updated it now. We use gradient buttons less than before, but that doesn't mean we won't use them at all.

Reply:
so everywhere we use the `DiagonalGradient` component without passing colors, we will have the primary color without gradient, right? this is what we want?